spec/system/placements/schools/placements/remove_a_placement_spec.rb

```diff
@@ -0,0 +1,102 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Schools / Placements / Remove a placement",
+               type: :system, service: :placements do
+  let!(:school) { create(:placements_school, name: "School 1", phase: "Primary") }
+  let!(:placement_1) { create(:placement, school:) }
+  let!(:placement_2) { create(:placement, school:) }
+  let!(:subject_1) { create(:subject, name: "Subject 1", subject_area: "primary") }
+  let!(:subject_2) { create(:subject, name: "Subject 2", subject_area: "primary") }
+
+  before do
+    given_i_sign_in_as_anne
+    given_a_placement_has_one_subject(placement_1, subject_1)
+    given_a_placement_has_one_subject(placement_2, subject_2)
+    when_i_visit_the_placement_show_page(placement_1)
+  end
```

Comment:
I think we should be able to achieve the same thing more simply, without needing a `given_a_placement_has_one_subject` method.

```suggestion
  let!(:placement_1) { create(:placement, school:, subjects: [subject_1]) }
  let!(:placement_2) { create(:placement, school:, subjects: [subject_2]) }
  let(:school) { build(:placements_school, name: "School 1", phase: "Primary") }
  let(:subject_1) { build(:subject, name: "Subject 1", subject_area: "primary") }
  let(:subject_2) { build(:subject, name: "Subject 2", subject_area: "primary") }

  before do
    given_i_sign_in_as_anne
    when_i_visit_the_placement_show_page(placement_1)
  end
```

That is:
- `let build()` dependent objects, and only `let! create()` the primary subjects of our tests
- provide `subjects: []` to the placement, which causes the join record to be created automatically for us

Reply:
resolved in [ca0bad9](https://github.com/DFE-Digital/itt-mentor-services/pull/334/commits/ca0bad985a2501c00c35bb99ee622bcf1a2f3790)